packages/k2-alpine/src/components/Alert/Alert.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { useState } from 'react'
+import { KeyboardTypeOptions } from 'react-native'
+import Dialog from 'react-native-dialog'
+import { Alert as NativeAlert } from 'react-native'
+
+export const AlertWithTextInputs = ({
+  visible,
+  title,
+  description,
+  verticalButtons,
```

Comment:
how does it look with verticalButtons?

Reply:
* verticalButtons: false
<img src=https://github.com/user-attachments/assets/5239523f-0671-421d-a756-73576a65cf3a width=320 />

* verticalButtons: true
<img src=https://github.com/user-attachments/assets/c6ed5979-8112-4cb5-9121-c7e02c258cc1 width=320 />

verticalButtons:true will be useful when alert has more than two buttons.